### PR TITLE
rm: CI LUKS encryption

### PR DESCRIPTION
**Pull request by ldts**
Following https://github.com/foundriesio/meta-lmp/pull/1530, we now allow users to define their own passphrases for CI LUKS encryption.

# PR Template and Checklist

Please complete as much as possible to speed up the reviewing process.

Readiness and adding reviewers as appropriate is required.

All PRs should be reviewed by a technical writer/documentation team and a peer.
If effecting customers—which is a majority of content changes—a member of Customer Success must also review.

## Readiness

* [ ] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

## Checklist

* [ ] Run spelling and grammar check, preferably with linter.
* [ ] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [ ] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [ ] Match tone and style of page/section.
* [ ] Run `make linkcheck`.
* [ ] View HTML in a browser to check rendering.
* [ ] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [ ] follow best practices for commits.
  * [ ] Descriptive title written in the imperative.
  * [ ] Include brief overview of QA steps taken.
  * [ ] Mention any related issues numbers.
  * [ ] End message with sign off/DCO line (`-s, --signoff`).
  * [ ] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [ ] Squash commits if needed.
* [ ] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 25f3ada46dc66592aada3b2a66ebb25b1243ab33 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2793/docs/artifacts/html/index.html

File: source/reference-manual/linux/linux-disk-encryption.rst [hunk 1]
```diff
@@ -11,9 +11,13 @@ LUKS to construct a symmetrically encrypted root file-system. The master
 key used for data encryption is protected and unlocked using a
 passphrase.
 
+Users can choose their own passphrase; if they don't, LmP will generate
+one for them.
```
Comment (kprosise) [inline source/reference-manual/linux/linux-disk-encryption.rst:15]:
```suggestion
You can choose your own passphrase; if you do not, LmP will generate one for you.
```

**Review by kprosise — APPROVED** [review]
Just a one suggestions with minor changes. Otherwise it LGTM.

**Comment by doanac** [comment]
Docs for de0dd548c19b860eb97c78c1d85087fea65f336c are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2795/docs/artifacts/html/index.html

**Review by vanmaegima — APPROVED** [review]
lgtm

**Review by MrCry0 — APPROVED** [review]

**Comment by ldts** [comment]
@kprosise jfyi

**Pull request merged**